x-pack/plugins/apm/public/components/shared/span_icon/span_icon.stories.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiCopy,
+  EuiPanel,
+  EuiSpacer,
+  EuiCodeBlock,
+} from '@elastic/eui';
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { EuiThemeProvider } from '../../../../../../../src/plugins/kibana_react/common';
+import { SpanIcon } from './index';
+import { typeIcons } from './get_span_icon';
+
+const types = Object.keys(typeIcons);
+
+storiesOf('shared/span_icon/span_icon', module)
```

Comment:
@sqren we have a story for node icons: https://github.com/smith/kibana/blob/b307cee64e3bee0cb7bcda742c3f94f17d5d8efe/x-pack/plugins/apm/public/components/app/service_map/__stories__/Cytoscape.stories.tsx#L73-L296, so maybe we should delete that one.

Also we should prefer using [Component Story Format](https://storybook.js.org/docs/react/api/csf) over the legacy API.

Reply:
From the Cytoscape story, I can't tell if we are show-casing the span icons (in which case it should just be deleted), or it's meant to show how it looks in the service map, in which case we should keep it.

It also looks like it's showing the agent icons via `iconForNode` so seems like it's covering more than the pure "span_icons" story.

Perhaps we can trim the cytoscape story down to not include every icon but just a few important ones.

Reply:
Okay, I see now that's it's simply a grid, similar to the one I made

![image](https://user-images.githubusercontent.com/209966/121964625-1b1d5000-cd6c-11eb-98a8-31184fcde9c6.png)

Reply:
Actually, notice how "aws" is empty. It works under the story I added:
![image](https://user-images.githubusercontent.com/209966/121964765-528bfc80-cd6c-11eb-87ec-98e143b84279.png)

There seems to be a bug here. Interesting!